### PR TITLE
(IMAGES-1271) Add template for Debian 11 

### PR DESCRIPTION
**Pull request by ciprianbadescu**

### CODE REVIEW

File: manifests/modules/packer/manifests/vmtools.pp [hunk 1]
```diff
@@ -14,7 +14,7 @@
   $vmtools_mountpoint = '/tmp/vmtools'
   # At some point it's going to become more worthwhile to flip this so
   # installing open-vm-tools is the default.
-  if ( ($facts['osfamily'] == 'debian' and $facts['operatingsystemmajrelease'] in ['7', '8', '9', '10', '16.04', '14.04', '18.04', '18.10', '20.04']) or
+  if ( ($facts['osfamily'] == 'debian' and $facts['operatingsystemmajrelease'] in ['7', '8', '9', '10', 'bullseye/sid', '16.04', '14.04', '18.04', '18.10', '20.04']) or
```
Comment (mihaibuzgau) [inline manifests/modules/packer/manifests/vmtools.pp:17]: This is temporary until Ubuntu 11 is released, right?

Comment (ciprianbadescu) [reply manifests/modules/packer/manifests/vmtools.pp:17]: right

File: templates/debian/11.0/x86_64/vars.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "template_name"                            : "debian-11-x86_64",
+    "template_os"                              : "debian10_64Guest",
```
Comment (gimmyxd) [inline templates/debian/11.0/x86_64/vars.json:3]: Should this be `debian11_64Guest`?

Comment (ciprianbadescu) [reply templates/debian/11.0/x86_64/vars.json:3]: Not yet available: https://code.vmware.com/apis/358/vsphere/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html

**Pull request merged**